### ISSUE TITLE
Error when running ZettelInbox

### ISSUE THREAD

**Issue by folofjc** [michal-h21/vim-zettel#127]
I have been using vim-zettel for a while, but have not yet tried `ZettelInbox`. I am running the latest of vim-zettel, and the `dev` branch on vimwiki. 

I get the following error when I run `:ZettelInbox`:
```
Error detected while processing function zettel#vimwiki#inbox:
line    6:
E119: Not enough arguments for function: vimwiki#base#check_links
```

If I run `:VimwikiCheckLinks` the command works fine.

**Comment by folofjc** [comment]
Update: When running on `master` of vimwiki, it works fine. Maybe they changed the syntax of the command on `dev`?

**Comment by michal-h21** [comment]
Yes, it seems that they changed the required parameters for `vimwiki#base#check_links` in the develoment version of Vimwiki. It should be fixed now. But for some reason, it doesn't work well with my Markdown test wiki. All links have double `.md.md` extension. Hopefully, it is issue only on my side.

**Comment by folofjc** [comment]
Ha. Yes, I noticed this and submitted [this issue about it already](https://github.com/vimwiki/vimwiki/issues/1256). They moved the `g:vimwiki_markdown_link_ext` variable to a per wiki variable and did not update the docs. Hopefully they will fix it on `dev` soon.

**Comment by michal-h21** [comment]
Ah, hopefully they will fix it soon.
